app/views/home/contributors.html.erb

```diff
@@ -1,13 +1,180 @@
+        </div>
+      </div>
+    </div>
+    <div class="float-left col-md-4">
+      <div class="clearfix mb-4 text-md-center">
+        <div class="float-left mr-4 float-md-none mr-md-0 mb-md-3"><img src="https://assets-cdn.github.com/images/modules/site/icons/opensource-ico-best.svg?sn" alt="" aria-hidden></div>
+        <div class="overflow-hidden">
+          <h3 class="alt-h4 mb-2">Work with the best in the&nbsp;field</h3>
+          <p class="alt-text-small text-gray">Amazing developers contribute to open source. Contribute code to projects that change how software&nbsp;is&nbsp;built.</p>
+        </div>
+      </div>
+    </div>
+    <div class="float-left col-md-4">
+      <div class="clearfix mb-4 text-md-center">
+        <div class="float-left mr-4 float-md-none mr-md-0 mb-md-3"><img src="https://assets-cdn.github.com/images/modules/site/icons/opensource-ico-grow.svg?sn" alt="" aria-hidden></div>
+        <div class="overflow-hidden">
+          <h3 class="alt-h4 mb-2">Grow your skills and help&nbsp;others</h3>
+          <p class="alt-text-small text-gray">Whatever your skill level, working on open source software is a great way to learn new&nbsp;things.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="bg-shade-gradient border-top py-6 px-3">
+  <div id="participate" class="container-md py-6">
+
+    <h2 class="alt-h2 text-center">Contribute on Open Source Friday</h2>
+    <p class="alt-lead text-gray text-center ">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+
+
+
+    <div class="d-md-table direction-md-rtl my-6 py-6 border-bottom">
+      <div class="d-md-table-cell v-align-middle col-md-6 text-center">
+        <a href="https://opensource.guide/how-to-contribute/">
+          <img alt="time illustration" class="img-responsive" src="">
+        </a>
+      </div>
+      <div class="d-md-table-cell v-align-middle col-md-6 direction-md-ltr pr-md-6">
+        <h3 class="alt-h3">Set aside time</h3>
+
+        <p class="text-gray mb-4">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        </p>
+
+        <!-- 2. Include script -->
+<script type="text/javascript">(function () {
+        if (window.addtocalendar)if(typeof window.addtocalendar.start == "function")return;
+        if (window.ifaddtocalendar == undefined) { window.ifaddtocalendar = 1;
+            var d = document, s = d.createElement('script'), g = 'getElementsByTagName';
+            s.type = 'text/javascript';s.charset = 'UTF-8';s.async = true;
+            s.src = ('https:' == window.location.protocol ? 'https' : 'http')+'://addtocalendar.com/atc/1.5/atc.min.js';
+            var h = d[g]('body')[0];h.appendChild(s); }})();
```

Comment:
I just copy/pasta'd it to get it working here. I'm not sure we want to keep this plugin, but if we do, we can move it out to an external script to the header.